tools/eslint-rules/prefer-common-mustnotcall.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview Prefer common.mustNotCall(msg) over common.mustCall(fn, 0)
+ * @author James M Snell <jasnell@gmail.com>
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const msg = 'Please use common.mustNotCall(msg) instead of ' +
+            'common.mustCall(fn, 0)';
+
+module.exports = function(context) {
+  return {
+    CallExpression(node) {
+      if (node.callee &&
+          node.callee.object &&
+          node.callee.object.name === 'common') {
+        if (node.callee.property &&
+            node.callee.property.name === 'mustCall') {
+          if (node.arguments.length >= 2) {
+            if (node.arguments[1].value == 0) {
```

Comment:
Could you at least break these out into one or more helper functions (`isCommonMethod()`, `isMustCall()`)?